src/form/bf_mx_lin.jl

```diff
@@ -24,6 +24,13 @@ function variable_mc_branch_power(pm::LPUBFDiagModel; nw::Int=nw_id_default, bou
 end
 
 
+"Capacitor switching and relaxed current variables"
```

Comment:
should correct to follow new conventions for docstring format that include the function signature